cluster/engine.go

```diff
@@ -921,8 +921,6 @@ func (e *Engine) CreateContainer(config *ContainerConfig, name string, pullImage
 	// Register the container immediately while waiting for a state refresh.
 	// Force a state refresh to pick up the newly created container.
 	e.refreshContainer(createResp.ID, true)
-	e.RefreshVolumes()
```

Comment:
For that volume manifest which is created by docker engine when creating container. Swarm event policy will make sure the volume manifest are refreshed. So I think `RefreshVolumes` in CreateContainer are somewhat redundant. 

```
func (e *Engine) handler(ev *dockerclient.Event, _ chan error, args ...interface{}) {
	// Something changed - refresh our internal state.

	switch ev.Type {
	case "network":
		e.RefreshNetworks()
	case "volume":
		e.RefreshVolumes()
```

@dongluochen